docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+services:
+  # This runs the database that everything else connects to
+  postgres-db:
+    image: postgres:11.10
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+      - POSTGRES_USER=${POSTGRES_USER:-postgres}
+      - POSTGRES_DB=${POSTGRES_DB:-postgres}
+    volumes:
+      - db-volume:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+    networks:
+      - default
```

Comment:
Is a default network really required if not for a particular use case?

Reply:
I don't think so